custom-recipes/timeseries-forecast-2-predict/recipe.json

```diff
@@ -134,7 +134,7 @@
             "defaultValue": 95,
             "minI": 1,
             "maxI": 99,
-            "description": "%"
+            "description": "%. Output lower and upper bounds forecasts."
```

Comment:
```suggestion
            "description": "Lower and upper confidence bounds in %"
```

Reply:
The "%" was meant to be right after the param:
<img width="650" alt="Screenshot 2020-12-21 at 19 01 33" src="https://user-images.githubusercontent.com/60091564/102807635-517ea180-43bf-11eb-8ac5-d1e4d9b4bcc1.png">
We could add it in the label " Confidence interval (in %)" and then have as description: "Compute lower and upper confidence bounds forecasts"

Reply:
Even better, yes!